Source/UserLocServer/Private/MovableCamera.cpp

```diff
@@ -130,6 +130,36 @@ float  AMovableCamera::TranslateY(float y_opencv) {
     return new_y;
 };
 
+
+/*
+    Function that calulates the roll of the camera.
+    Takes the change in x position. 
+
+    Returns the new roll
+
+*/
+float AMovableCamera::rotation_roll(float x_change) {
+    float droll = 0.09; 
+    float c = 15;
+
+    return droll * (x_change - WidthUE / 2) + c;
+};
+
+
+/*
+    Function that calulates the pitch of the camera.
+    Takes the change in y position.
+
+    Returns the new pitch
+
+*/
+float AMovableCamera::rotation_pitch(float y_change) {
+    float dpitch = 0.06;
+    float c = 15;
```

Comment:
dPitch can be a configurable variable @haugeSander 

default value should be 0.06